pkg/checks/external/nodeCheck/main.go

```diff
@@ -79,21 +79,25 @@ func waitForKuberhealthyEndpointReady(ctx context.Context, kuberhealthyEndpoint
 	doneChan := make(chan error, 1)
 
 	for {
+		res, err := http.Get(kuberhealthyEndpoint)
+		if err != nil {
+			log.Debugln(kuberhealthyEndpoint, "Error making request to kuberhealthy endpoint "+err.Error())
+			continue
+		}
+		if res.StatusCode == 200 {
```

Comment:
possibly we don't need a 200, just getting a response/err being nil is enough?

Reply:
I think technically we could still get a non-200 response and a non-nil error. So checking that we get an healthy response is appropriate still.